Consul/Client.cs

```diff
@@ -595,67 +595,67 @@ void ApplyConfig(ConsulClientConfiguration config, HttpClientHandler handler, Ht
 
         internal GetRequest<TOut> Get<TOut>(string path, QueryOptions opts = null, IEncodable filter = null)
         {
-            return new GetRequest<TOut>(this, path, opts ?? QueryOptions.Default, filter);
```

Comment:
`?? QueryOptions.Default` is not necessary here as it is going to be done in the constructor of `GetRequest`